src/js/modules/domain/generatedRpc/entitiesDefinition.json

```diff
@@ -176,15 +176,15 @@
       "fields": [
```

Comment:
please be disambiguate your commit subject line by explaining what "correct type" is. i think what is happening here is that the types were for kafka wire protocol, but coproc is using non-varint format found in the internal redpanda rpc protocol. but that's just a guess. the commit message should explain it.

Reply:
Got it, sounds good